### PR TITLE
made footer

### PR DESCRIPTION
**Pull request by Zoyakhatry1**

### CODE REVIEW

**Review by irffanasiff — COMMENTED** [review]
@Zoyakhatry1 you made some changes to index.html please check that try only pushing the code you have written please don't change the previous code

File: index.html [hunk 1]
```diff
@@ -2,56 +2,54 @@
      ALSO DO NOT MAKE ANY CHANGES TO other css files YOU HAVE TO ONLY WORK ON style.css-->
 <!DOCTYPE html>
 <html lang="en" xmlns="http://www.w3.org/1999/xhtml">
```
Comment (irffanasiff) [inline index.html:4]: please check here

File: index.html [hunk 2]
```diff
@@ -2,56 +2,54 @@
...
-    <!-- Font Awesome -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
-  
-    <!--CSS3 Files-->
 
-    
-    <link rel="stylesheet" href="CSS\style.css" />
-    <link rel="stylesheet" href="CSS\style-blur-navbar.css" />
-    <link rel="stylesheet" href="CSS\hero_section.css" />
-  </head>
```
Comment (irffanasiff) [inline index.html:46]: and here

File: index.html (hunk 1 above)
Comment (Zoyakhatry1) [reply index.html:4]: okay...

**Pull request closed without merging**